src/utils/DockerMachineUtil.js

```diff
@@ -85,9 +85,9 @@ var DockerMachine = {
   regenerateCerts: function (machineName = this.name()) {
     return util.exec([this.command(), 'tls-regenerate-certs', '-f', machineName]);
   },
-  state: function (machineName = this.name()) {
-    return this.info(machineName).then(info => {
-      return info ? info.state : null;
+  status: function (machineName = this.name()) {
```

Comment:
No big deal but this seems to be from another change - is master merged into this?